PlayerUI/Util/NSWindow+Snapshot.swift

```diff
@@ -9,7 +9,6 @@
 import Cocoa
 
 extension NSWindow {
-
```

Comment:
I disagree with many removals of the vertical whitespace. Vertical whitespace can greatly improve readability.